### ISSUE TITLE
Can I disable swipe after reaching index 1 ??

### ISSUE THREAD

**Issue by GaneshShetty951** [juanpablofernandez/SwiftyOnboard#22]

**Comment by jdanthinne** [comment]
Really interested in this as well. I'd like to make a step to accept notifications, and until notifications are not accepted, disable swiping to next slide.

**Comment by GaneshShetty951** [comment]
@jdanthinne i did it without the notification but using notifications sounds better

**Comment by GaneshShetty951** [comment]
`swifty_Onboard?.containerView.isScrollEnabled = false`

I wrote this line but don't know how much good practice it is. @jdanthinne

**Comment by jdanthinne** [comment]
`'containerView' is inaccessible due to 'fileprivate' protection level`

**Comment by GaneshShetty951** [comment]
@jdanthinne I removed the > fileprivate

**Comment by jdanthinne** [comment]
Yes, of course, but I was using the Carthage version, so I was wondering if there was an official way to do it…

**Comment by GaneshShetty951** [comment]
@jdanthinne Sorry about that, we can create method to enable and disable inside that file also if we don't want to remove `fileprivate`. right ??

**Comment by jdanthinne** [comment]
Exactly, or the ability to change the `shouldSwipe` var after setup…
